
geometry_msgs/Vector3 pos
geometry_msgs/Vector3 ori
geometry_msgs/Vector3 vel
geometry_msgs/Vector3 dot_ori
geometry_msgs/Vector3 acc
geometry_msgs/Vector3 dotdot_ori
